contracts/smart-contract-wallet/BaseSmartAccount.sol

```diff
@@ -10,6 +10,7 @@ import {IEntryPoint} from "@account-abstraction/contracts/interfaces/IEntryPoint
 import {UserOperationLib, UserOperation} from "@account-abstraction/contracts/interfaces/UserOperation.sol";
 import {Enum} from "./common/Enum.sol";
 import {BaseSmartAccountErrors} from "./common/Errors.sol";
+import "@account-abstraction/contracts/core/Helpers.sol";
```

Comment:
should use the named imports as per the QA done

Reply:
would have been too many named imports as this file is a library